x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -25,7 +25,7 @@ export const DEFAULT_TIME_RANGE = 'timepicker:timeDefaults';
 export const DEFAULT_REFRESH_RATE_INTERVAL = 'timepicker:refreshIntervalDefaults';
 export const DEFAULT_APP_TIME_RANGE = 'securitySolution:timeDefaults';
 export const DEFAULT_APP_REFRESH_INTERVAL = 'securitySolution:refreshIntervalDefaults';
-export const DEFAULT_ALERTS_INDEX = '.alerts-security-solution';
+export const DEFAULT_ALERTS_INDEX = '.alerts-security.alerts';
```

Comment:
At some point we should make the data structure here a kibana package https://github.com/elastic/kibana/blob/e2aacdc186c04964dcbc0ce81a7b2a25dbd8a83a/x-pack/plugins/rule_registry/server/utils/rbac.ts#L16-L23 or possible a saved object that stores all of this information.. No changes here just another note.

Reply:
I actually moved it into it's own package in this PR. But yea, definitely could be improved even further.